components/konflux-ci/base/external-secrets/konflux-ci-repo-creator.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: konflux-ci-repo-creator
+  annotations:
+    argocd.argoproj.io/sync-options: SkipDryRunOnMissingResource=true
+    argocd.argoproj.io/sync-wave: "-1"
+spec:
+  dataFrom:
+    - extract:
+        key: production/build/konflux-ci-repo-creator
```

Comment:
secrets which are applied to `konflux-ci` namespace are under directory `production/build/tekton-ci/`, will it be good to move it under `tekton-ci` .. (tough the directory name not matching, we may try to rename it in future)